site/_docs/provisioning/quickstart.md

```diff
@@ -37,20 +38,36 @@ NB. the instructions will also work in anywhere supported by [Docker Machine](ht
                 - your AWS key and your secret access key (see [getting your AWS access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html)) via command line options, environment variables or using aws configure 
                 - the AWS region id in this format: eu-west-1
             - a username and password (optional) to act as credentials for the initial admin user (you will be prompted to re-enter your password if it is considered weak)
-
+                - The initial admin username cannot be set to 'admin' to avoid duplicate entries in LDAP.
+            - AWS parameters i.e. a subnet ID, the name of a keypair and an EC2 instance type (these parameters are useful if you would like to extend the platform with additional AWS EC2 services)
     - For example (if you don't have ~/.aws set up):
 
         ```./quickstart.sh -t aws -m adop1 -a AAA -s BBB -c vpc-123abc -r eu-west-1 -u user.name -p userPassword```
-
         - N.B. If you see an error saying that docker-machine cannot find an associated subnet in a zone, go back to the VPC Dashboard on AWS and check the availablity zone for the subnet you've created. Then rerun the startup script and use the -z option to specify the zone for your subnet, e.g. for a zone of eu-west-1c the above command becomes:
 
             ```./quickstart.sh -t aws -m adop1 -a AAA -s BBB -c vpc-123abc -r eu-west-1 -u user.name -p userPassword -z c```
+5. If all goes well you will see the following output and you can view the DevOps Platform in your browser
+    ```
+    ##########################################################
+
+    SUCCESS, your new ADOP instance is ready!
 
-1. If all goes well you will see the following output and you can view the DevOps Platform in your browser using the link that accompanies this output:
+    Run this command in your shell:
+      source ./conf/env.provider.sh
+      source credentials.generate.sh
+      source env.config.sh
+      
+    You can check if any variables are missing with: ./adop compose config  | grep 'WARNING'
 
-    ```SUCCESS, your new ADOP instance is ready!```
+    Navigate to http://11.22.33.44 in your browser to use your new DevOps Platform!
```

Comment:
Is this IP address something which everyone have by default or it's just an example? Maybe worth to replace on "Navigate to http://<proxy_ip>" ?